package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua

```diff
@@ -1,8 +1,14 @@
 return function(form, uci)
-	if uci:get_bool("autoupdater", "settings", "enabled") then
+	local enabled = uci:get_bool("autoupdater", "settings", "enabled")
+	if enabled then
 		local s = form:section(
 			Section, nil,
 			translate('This node will automatically update its firmware when a new version is available.')
 		)
+	else
+		local s = form:section(
+			Section, nil,
+			translate('Automatic updates are disabled. They can be enabled in <em>Advanced settings</em>')
+		)
```

Comment:
Assigning the section to a local variable doesn't make sense here... while at it, you can also fix that for the branch you didn't touch.